eth/bind.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
This file is entirely unnecessary, use package ethclient instead (see review comment)